libgdiplus will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libgdiplus it will fail in the following way:

```
2024/11/12 23:11:56 INFO running step "test/pkgconf"
2024/11/12 23:11:56 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:11:56 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:11:56 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:11:56 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:11:57 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN + basename /home/build/melange-out/libgdiplus-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN + dev_pkg=libgdiplus-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN + apk info -L libgdiplus-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN + grep -q ^Name: usr/lib/pkgconfig/libgdiplus.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN + basename usr/lib/pkgconfig/libgdiplus.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN + lib_name=libgdiplus uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN + echo usr/lib/pkgconfig/libgdiplus.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN + grep -q '^usr/lib/pkgconfig/libgdiplus.pc$\|^usr/share/pkgconfig/libgdiplus.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN + pkgconf --exists libgdiplus uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN + pkgconf --print-errors --exists libgdiplus uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN Package glib-2.0 was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN Perhaps you should add the directory containing `glib-2.0.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN Package 'glib-2.0', required by 'libgdiplus', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN Package 'gmodule-2.0', required by 'libgdiplus', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 WARN Package 'gthread-2.0', required by 'libgdiplus', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:57 INFO pod 55e533189f5a3a90f0a26df229bfed2006243e006005662acdc416e4527ba147 terminated
2024/11/12 23:11:57 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:11:57 INFO   workspace dir: /home/user/tmp/melange-workspace-2206461246
2024/11/12 23:11:57 INFO   guest dir: /home/user/tmp/melange-guest-3364659943
2024/11/12 23:11:57 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libgdiplus] Error 1
```

--- a/libgdiplus.yaml
+++ b/libgdiplus.yaml
@@ -1,7 +1,7 @@
 package:
   name: libgdiplus
   version: 6.1
-  epoch: 1
+  epoch: 2
   description: "Open Source Implementation of the GDI+ API"
   copyright:
     - license: MIT
@@ -58,8 +58,12 @@ subpackages:
       - uses: split/dev
     dependencies:
       runtime:
+        - glib-dev
         - libgdiplus
     description: libgdiplus dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
 update:
   enabled: true

--- a/libgdiplus.yaml
+++ b/libgdiplus.yaml
@@ -58,7 +58,6 @@ subpackages:
       - uses: split/dev
     dependencies:
       runtime:
-        - glib-dev
         - libgdiplus
     description: libgdiplus dev
     test:

